lib/facets/aggs-all.js

```diff
@@ -81,43 +81,33 @@ module.exports = function (queryParams) {
 function filter (queryParams, exclude) {
   var filters = [];
   // category
-  if (exclude !== 'category') {
-    const categories = queryParams.filter.objects.categories;
-    if (categories) {
-      filters.push({ terms: { 'categories.value': categories } });
-    }
+  const categories = queryParams.filter.objects.categories;
```

Comment:
is `queryParams.filter.objects` _always_ defined? 
or _could_ you get a `TypeError` here ? https://repl.it/Cb7H